code/_onclick/hud/_defines.dm

```diff
@@ -194,3 +194,17 @@
 #define ui_adminbus_tdarena "2:4,13:26"
 #define ui_adminbus_tdgreen "3:6,13:26"
 #define ui_adminbus_tdobs "2:4,14:28"
+
+//Blob HUD
+#define ui_blob_bg "1:0,1:0"
```

Comment:
Both these and the adminbus UIs need to use the system used above in the rest of the defines here.

Byond defines the corners of your screen as NORTH, SOUTH, EAST, WEST in the screen locs. By defining with NORTH,SOUTH,EAST,WEST your hud will automatically scale with the size of your screen.

So to align somethign with the bottom corner you make the screen loc "EAST,SOUTH", this is compatible with any other location through using EAST+x.